src/components/pages/RoleApply/Styles/apply.css

```diff
@@ -32,11 +32,7 @@
 
 .card-img {
   max-height: 70vh;
-  max-width: 90vw;
-}
-
-.card1 .card-img {
-  width: 98.2%;
+  max-width: 100%;
```

Comment:
This is slightly overkill as typically items will not have a width of larger than 100%. width is fine here